cblox/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ cs_add_library(${PROJECT_NAME}_lib
   src/mesh/submap_mesher.cpp
   src/io/transformation_io.cpp
 )
-target_link_libraries(${PROJECT_NAME}_lib ${catkin_LIBRARIES} ${PROJECT_NAME}_proto)
+target_link_libraries(${PROJECT_NAME}_lib ${catkin_LIBRARIES} ${PROJECT_NAME}_proto) # -fsanitize=address,leak
```

Comment:
I guess you don't need `${catkin_LIBRARIES}`, this should be automatically linked using catkin simple.
Also, rm comments here and up? :)

Reply:
ahh, cheers.